source/integrations/cloud-outgoing-webhooks.rst

```diff
@@ -48,7 +48,7 @@ In our example, we would set the channel to ``town-square`` and specify ``#build
  - If the first word of a message matches one of the trigger words exactly, or
  - If the first word of a message starts with one of the trigger words.
 
-6. Finally, set one or more callback URLs that HTTP POST requests will be sent to, and select **Save**. If the URL is private, add it as a `trusted internal connection <https://about.mattermost.com/default-allow-internal-connections-settings-documentation/>`__.
+6. Finally, set one or more callback URLs that HTTP POST requests will be sent to, and select **Save**. If the URL is private, add it as a `trusted internal connection <https://docs.mattermost.com/configure/configuration-settings.html>`__.
```

Comment:
```suggestion
6. Finally, set one or more callback URLs that HTTP POST requests will be sent to, and select **Save**. If the URL is private, add it as a `trusted internal connection <https://docs.mattermost.com/configure/configuration-settings.html#allow-untrusted-internal-connections-to>`__.
```

Reply:
I think this may be the correct link for this page.

Reply:
Yeah, that permalink looks better and heads over to the specific section we want to look at. Thanks for the addition.